scripts/install_oxide

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e
+
+CFU_ROOT="$(dirname $(dirname $(realpath ${BASH_SOURCE[0]})))"
+
+
+UPDATE=
+if [ $# -gt 1 ] ; then
+  echo "Usage: install_oxide [update]"
+  exit 1
+fi
+if [ $# -gt 0 ] ; then
+  if [ $1 == "update" ] ; then
+    UPDATE=1
+  else
+    echo "Usage: install_oxide [update]"
+    exit 1
+  fi
+fi
+
+
+
+if [ -z "$UPDATE" ] ; then
+
+  echo -n "Install Project Oxide open source toolchain? (ctrl-C to exit)" >&2
```

Comment:
Did you mean to put a `read` command or something after this?

Reply:
Correct.   I will just delete this, since it's not changing system state (as in, installing apt packages).